src/main/scala/inox/transformers/SimplifierWithPC.scala

```diff
@@ -132,7 +132,9 @@ trait SimplifierWithPC extends TransformerWithPC { self =>
       }
     }
 
-    def withCond(e: Expr) = if (formulaSize(e) > 20) this else {
+    override def withBound(b: ValDef) = ???
```

Comment:
I don't know how to implement this. But it doesn't seems to bother runtime, at least not in inox test suite.

Reply:
We don't need to keep track of bound variables in `CNFPath`, but please return `this` instead of `???` to avoid crashing if we encounter a `Forall` or something.

Reply:
done.